src/reanimated2/animation/spring.ts

```diff
@@ -138,11 +137,12 @@ export function withSpring(
           (previousAnimation?.startValue as number)
         : Number(animation.toValue) - value;
 
-      animation.velocity = previousAnimation
-        ? triggeredTwice
-          ? previousAnimation.velocity
-          : previousAnimation.velocity + config.velocity
-        : config.velocity;
+      animation.velocity =
+        (previousAnimation
+          ? triggeredTwice
+            ? previousAnimation?.velocity
+            : previousAnimation.velocity + config.velocity
+          : config.velocity) || 0;
```

Comment:
Can we rewrite it to standard `if ()` syntax? because this one is complicated to read